docs/glossary.md

```diff
@@ -92,7 +92,7 @@ Dexterity
     [Dexterity](https://github.com/plone/plone.dexterity), the base framework for building content types, both through-the-web and as filesystem code for Zope.
 
 Dublin Core
-    The Dublin Core Schema is a small set of vocabulary terms that can be used to describe web resources (video, images, web pages, etc.), as well as physical resources such as books or CDs, and objects like artworks.
+    The Dublin Core Schema is a small set of vocabulary terms that can be used to describe web resources (video, images, web pages, etc.). It can also be used to describe physical resources such as books or CDs, and objects like artworks.
```

Comment:
This follows both ours and the Microsoft Style Guide more closely, avoiding both the use of `etc.` and using the correct and of a sentence for a list of items. See https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/and-so-on

```suggestion
    The Dublin Core Schema is a small set of vocabulary terms that can be used to describe web resources (video, images, web pages, and other online content).
    It can also be used to describe physical resources, such as books, CDs, artworks, and other objects.
```